src/inbox/mod_inbox_rdbms_mssql.erl

```diff
@@ -49,10 +37,7 @@ set_inbox_incr_unread(Username, Server, ToBareJid, Content, MsgId, Timestamp) ->
 build_query(Username, Server, ToBareJid, Content, increment, MsgId, Timestamp) ->
     CountUpdate = "target.unread_count + 1",
     build_query(Username, Server, ToBareJid, Content, MsgId,
-                Timestamp, esc_string("1"), CountUpdate);
-build_query(Username, Server, ToBareJid, Content, CountValue, MsgId, Timestamp) ->
-    ECount = esc_string(CountValue),
-    build_query(Username, Server, ToBareJid, Content, MsgId, Timestamp, ECount, ECount).
+                Timestamp, esc_string("1"), CountUpdate).
 
 build_query(Username, Server, ToBareJid, Content, MsgId, Timestamp, CountInsert, CountUpdate) ->
```

Comment:
I think it would be nice to have a one line comment here, why this function does not reuse generic upsert helper.

Reply:
I instead commented on the generic `mod_inbox_rdbms:set_inbox_incr_unread`.